modules/ossm-control-plane-deploy.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/service_mesh_install/installing-ossm.adoc
+
+[id="ossm-control-plane-deploy_{context}"]
+
+= Deploying the control plane
+
+With the introduction of {product-title} 4.1, the network capabilities of the host are now based on nftables rather than iptables. This change impacts the initialization of the {ProductShortName} application components. {ProductShortName} needs to know what host operating system OpenShift is running on to correctly initialize {ProductShortName} networking components.
+
+If the OpenShift installation is deployed on a Red Hat Enterprise Linux (RHEL) 7 host, then the custom resource must explicitly request the RHEL 7 proxy-init container image by including the following:
+
+.Enabling the proxy-init container for RHEL 7 hosts
+
+[source,yaml]
+----
+  apiVersion: maistra.io/v1
+   kind: ServiceMeshControlPlane
+   spec:
+     istio:
+       global:
+         proxy_init:
+           image: proxy-init
+----
+
+
+Use the custom resource definition file you created to deploy the {ProductShortName} control plane.
+
+
+.Procedure
+
+. Run this command to deploy the control plane:
++
+----
+$ oc create -n istio-system -f istio-installation.yaml
+----
++
+The operator creates the `istio-system` namespace and runs the installer job; this job installs and configures the control plane. You can follow the progress of the installation by watching the pods.
```

Comment:
Operator and all kubernetes constructs should always be capitalized.

Reply:
Fixed.